src/main/java/hu/elte/eserial/annotation/ExcludeThis.java

```diff
@@ -0,0 +1,16 @@
+package hu.elte.eserial.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import hu.elte.eserial.annotation.enumeration.EserialAnnotationType;
+
+/**
+ * Indicates that an element should be excluded from the serialization/deserialization process.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD, ElementType.METHOD})
+@EserialAnnotation(priority = Integer.MAX_VALUE, type = EserialAnnotationType.Inclusion)
```

Comment:
I understand that this annotation should have the highest priority, but is Integer.MAX_VALUE a good decision? I think a high value should be also fine, and it wouldn't have to be modified in the future if a new annotation with higher priority arrives.

Reply:
My thought process was that there shouldn't be any annotation which overwrites the effect of the `@ExcludeThis` annotation. But of course it can be lowered. Which values do you recommend for the default annotations?

Currently these are set:
* `ExcludeThis`: max,
* `IncludeElements`: 100
* `IncludeMatching`: 10

Reply:
If you are 100% sure that ExcludeThis will always be the highest priority, then max is viable. Otherwise priority = 1000?

Reply:
Or I can make them linearly increasing, like a normal person: 1000/2000/3000. I don't know why I started with exponentially increasing values. 

Having more flexibility is always better, so I won't use Integer.MAX_VALUE.